norona_marynicole/index.html

```diff
@@ -0,0 +1,187 @@
+        <li>Dancing Tiktok</li>
+      </ul>
+    </div>
+
+    <div>
+      <h2 class="title">MY WORKS</h2>
+      <ol>
+        <li><a href="1.html">html</a></li>
+        <li><a href="2.html">html</a></li>
+        <li><a href="3.html">html</a></li>
+        <li><a href="4.html">html</a></li>
+        <li><a href="5.html">html</a></li>
+      </ol>
+    </div>
+
+    <div>
+      <h2 class="goals">MY GOALS</h2>
+      <ol>
+        <li>To finish my studies</li>
+        <li>To have a stable and great job</li>
+        <li>To be financially stable</li>
+        <li>To give a comfortable life to my family</li>
+        <li>To be rich and successful in life</li>
+      </ol>
+
+      <h2 class="title">COMMENTS OF MY TEAMMATES</h2>
+      <ul>
+        <li>
+          <p>
+            Your goals for education, stability, financial security, and family
+            well-being reflect ambition and determination, promising a future of
+            success and fulfillment. - Bergado, Shamma
+          </p>
+        </li>
+        <li>
+          <p>
+            Your goals perfectly represent your values and desires. I'm thrilled
+            to see how you bring them to life! - Morcoso, Daisy
+          </p>
+        </li>
+        <li>
+          <p>
+            Dream big, work hard, and never lose sight of your goals; with
+            dedication and perseverance, you'll achieve greatness. -Reyes, John
+            Ruiz.
+          </p>
+        </li>
+        <li>You can do it! -Reyes, Paul Angelo</li>
+      </ul>
+    </div>
+
+    <div>
+      <h2 class="comment">COMMENTS ON ME:</h2>
+    </div>
+
+    <form class="comment" id="comment">
+      <div>
+        <label for="name">Name: </label>
+        <input type="text" id="name" />
+      </div>
+      <div>
+        <p>Comment:</p>
+        <textarea name="gcomment" id="gcomment" cols="30" rows="10"></textarea>
+      </div>
+      <div>
+        <button type="button" id="comment_button">COMMENT</button>
+        <button type="button" id="sort_asc" onclick="sortComments('asc')">
+          Sort by Date (Ascending)
+        </button>
+        <button type="button" id="sort_desc" onclick="sortComments('desc')">
+          Sort by Date (Descending)
+        </button>
+      </div>
+    </form>
+
+    <div>
+      <h2 class="opinion">MY OWN OPINION ON F2F OR ONLINE CLASS</h2>
+      <p>
+        In my perspective, conducting a face-to-face class last April 20, 2024,
+        was more convenient for understanding the lesson easily. It was
+        advantageous since not all students have necessary materials like
+        laptops, PCs, and internet access at home, especially for subjects like
+        coding. Face-to-face classes facilitate better understanding of
+        practical activities and provide necessary materials like PCs and
+        internet access. However, in deciding between face-to-face or online
+        classes, safety should be a priority, especially considering the current
```

Comment:
There should be max of 80 col per line.

Follow the guidelines
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_0